src/main/java/org/skywalking/apm/ui/creator/ImageBase64Creator.java

```diff
@@ -89,11 +88,10 @@ private String getImageFolder() {
     }
 
     private String imageEncode(byte[] imageData, String imageType) {
-        BASE64Encoder encoder = new BASE64Encoder();
         if (PNG.equals(imageType)) {
-            return PNG_BASE64_PREFIX + encoder.encode(imageData);
+            return PNG_BASE64_PREFIX + Base64.getEncoder().encode(imageData);
         } else {
-            return JPG_BASE64_PREFIX + encoder.encode(imageData);
+            return JPG_BASE64_PREFIX + Base64.getEncoder().encode(imageData);
```

Comment:
从JDK官方注释已经说明, encoder和decoder对象是允许多线程使用的，即线程安全的。

```java
    /**
     * This class implements an encoder for encoding byte data using
     * the Base64 encoding scheme as specified in RFC 4648 and RFC 2045.
     *
     * <p> Instances of {@link Encoder} class are safe for use by
     * multiple concurrent threads.
     *
     * <p> Unless otherwise noted, passing a {@code null} argument to
     * a method of this class will cause a
     * {@link java.lang.NullPointerException NullPointerException} to
     * be thrown.
     *
     * @see     Decoder
     * @since   1.8
     */
    public static class Encoder {
```